CONTRIBUTING.md

```diff
@@ -568,7 +571,15 @@ There are three types of assets you can build:
 
 #### Webpack dev server
 
-The dev server by default starts at `http://localhost:9000` and proxies the backend requests to `http://localhost:8088`. It's possible to change these settings:
+The dev server by default starts at `http://localhost:9000` and proxies the backend requests to `http://localhost:8088`.
+
+Concretely, it means that you will want to:
+
+1. [run Superset locally](#flask-server) using Flask, on port `8088`,
+2. in parallel, run the Webpack dev server locally on port `9000`,
```

Comment:
Not sure if this is a good place to mention that you can run `npm run dev` or (what most people do) `npm run dev-server` for hot reloading

Reply:
Thanks! Would [that](https://github.com/apache/superset/pull/19870/commits/94aab8f6815e9f76d3e58bfde911d7bf33547624) make things clearer?